src/main/java/io/reactivex/disposables/SubscriptionDisposable.java

```diff
@@ -15,6 +15,9 @@
 import org.reactivestreams.Subscription;
 
 final class SubscriptionDisposable extends ReferenceDisposable<Subscription> {
+    /** */
+    private static final long serialVersionUID = -707001650852963139L;
```

Comment:
Eclipse complains about serializable classes (i.e., extending AtomicX) need these fields, even though we don't care about serialization. A small inconvenience.